examples/paho-mqtt/README.md

```diff
@@ -0,0 +1,44 @@
+### About
+This application demonstrates the usage of the MQTT paho package in RIOT.
+
+### Setup
+For using this example, two prerequisites have to be fulfilled:
+
+1. You need a running MQTT broker like Mosquitto broker for example. Take a look at
+[Mosquitto Broker](https://mosquitto.org/). Check online any guide that will help you setting up the broker into some port (a).
+For example this one for debian base linux users [How to setup a Mosquitto MQTT Server and receive data](https://www.digitalocean.com/community/questions/how-to-setup-a-mosquitto-mqtt-server-and-receive-data-from-owntracks).
+
+2. Your RIOT node needs to be able to speak to that broker at the same port you set in 1.
+
+### Setting up RIOT `native` on Linux
+- Run `sudo ./dist/tools/tapsetup/tapsetup -c 1`
+
+## Running the example
+- Run on RIOT's root directory:
+
+  make -C examples/paho-mqtt all term
+
+- To connect to a broker, use the `con` command:
+```
+con <ipv6 addr> <port> <KeepAliveInterval in sec> <user> <user ID> <password>
```

Comment:
```suggestion
con <ipv6 addr> <port> <KeepAliveInterval in sec> <client ID> <user> <password>
```

Reply:
`    data.username.cstring = argv[4];
    data.clientID.cstring = argv[5];
    data.password.cstring = argv[6];` regarding what I wrote, that are not the right positions, and I think client ID is not so important as username and password. I'll  s/user ID/client ID/ though

Reply:
Any ways, I think the order clientID user pass, it's good for a better undertandig